website/src/components/wordpress.tsx

```diff
@@ -87,10 +107,81 @@ const parseOptions: HTMLReactParserOptions = {
   },
 }
 
-const pullWords = (
-  docName: string,
-  start: string,
-  end?: string
-): JSX.Element => {
-  return <div>full array</div>
+const PullWords = (props: { slug: string; start: number; end?: number }) => {
+  const slug = props.slug
+  const start = props.start
+
+  var end
+  if (!props.end) {
+    end = start + 1
+  } else {
+    end = props.end
+  }
```

Comment:
Check if you can condense this to `end = props.end ? props.end : start + 1` or something similar.